src/main/java/com/ciandt/techgallery/service/transformer/TechnologyTransformer.java

```diff
@@ -25,6 +24,7 @@ public Technology transformFrom(TechnologyTO baseObject) {
     product.setEndorsersCounter(baseObject.getEndorsersCounter());
```

Comment:
@tsr-thulio Seems this method has the same content as the block that is on line 34